src/main/java/org/jenkinsci/plugins/vsphere/builders/ReconfigureCpu.java

```diff
@@ -34,11 +35,16 @@
 
     private final String cpuCores;
     private final String coresPerSocket;
+    private final String cpuLimitMHz;
+    private final ResourceAllocationInfo cpuReservation;
 
 	@DataBoundConstructor
-	public ReconfigureCpu(String cpuCores, String coresPerSocket) throws VSphereException {
+	public ReconfigureCpu(String cpuCores, String coresPerSocket, String cpuLimitMHz) throws VSphereException {
```

Comment:
Don't change the constructor - that can break things for existing users, especially folks who're using pipeline builds.
Create a `@DataBoundSetter` method `setCpuLimitMHz()` instead.
(Yes, this will mean that `cpuLimitMHz` can't be final - just put a comment in saying `/* almost final */`)

FYI while this plugin pre-dates such a practice being widely known, these days it's considered Jenkins "best practice" to have a constructor that only take mandatory arguments (and to have as few of those as possible) with everything else being set by data bound set methods.  This makes it much easier for code that doesn't know or care about newly introduced optional fields to continue to work.